src/Features/Core/Portable/GenerateConstructorFromMembers/AbstractGenerateConstructorFromMembersCodeRefactoringProvider.cs

```diff
@@ -229,11 +229,16 @@ public async Task<ImmutableArray<IntentProcessorResult>> ComputeIntentAsync(Docu
             var canAddNullCheck = viableMembers.Any(
                 static m => m.GetSymbolType().CanAddNullCheck());
 
+            var globalOptions = document.Project.Solution.Services.GetService<ILegacyGlobalOptionsWorkspaceService>();
```

Comment:
This check is slightly strict because the service is only used when `canAddNullCheck` is true.

I make this because the provided code action here is CodeActionWithDialog, which is something depends on IPickMemberService, and should not be provided in LSP.

Ideally, we also need to check IPickMemberService here, but that needs to modify the MEF, and refactor all the test here to use new test framework.
Would be a pretty large PR, so I prefer to do it later